src/configurations/nf/synapseConfigs/studies.ts

```diff
@@ -54,6 +54,21 @@ export const studyCardConfiguration: CardConfiguration = {
     baseURL: 'Explore/Studies/DetailsPage',
     URLColumnName: 'studyId',
     matchColumnName: 'studyId',
+  },
+  columnIconOptions: {
+    iconBasePath: "/icons/",
```

Comment:
May want to pull this out as a constant (make things easier if we ever want to move things around).

Reply:
I added a file `src/utils/PortalsConstants.ts` to save all the constants for portals.